docs/source/setup.rst

```diff
@@ -174,3 +174,21 @@ This is typically enforced through external means, such as by having a network
 file system (NFS) mount for libraries, by starting the ``dask-worker``
 processes in equivalent docker containers, or through any of the other means
 typically employed by cluster administrators.
+
+Windows
+~~~~~~~
+
+.. note::
+
+  - Running a ``dask-scheduler`` on Windows architectures is supported for only a
+    limited number of workers (roughly 100). This is a detail of the underlying tcp server
+    implementation and is discussed `here`__.
```

Comment:
In case it's not clear - I'm happy for this PR to go in as it's an accurate description of where we are today with tornado/Python and gives a workaround.